source/funkin/editors/charter/Charter.hx

```diff
@@ -225,12 +227,9 @@ class Charter extends UIState {
 					},
 					null,
 					{
-						label: "(0) Default Note",
-						keybind: [ZERO]
-					},
-					{
-						label: "(1) Hurt Note",
-						keybind: [ONE]
+						label: "Edit note types",
+						keybind: [],
```

Comment:
You should make it so it says

[0] default
[1] hurt
[2] death
...

And like so when you press a number key it switches.

Or like make it so

[0] default
[1] go to next note type
[2] go to previous note type